### PR TITLE
Feature | gds assessment | select workplace page

### PR DESCRIPTION
**Pull request by omarCreativeDev**
- made `select-workplace` page GDS compliant

### CODE REVIEW

File: src/app/features/select-workplace/select-workplace.component.html [hunk 1]
```diff
@@ -1,56 +1,54 @@
...
 
+      <p>
+        <a
+          [routerLink]="['/select-workplace-address']"
+          (click)="workplaceNotFound()"
+          title="My workplace isn’t displayed">
+          My workplace isn’t displayed
+        </a>
+      </p>
+      <button class="govuk-button govuk-!-margin-top-2" type="submit">Continue</button>
```
Comment (ChazUK) [inline src/app/features/select-workplace/select-workplace.component.html]: I know you've put it on most of the other Submit / Continue buttons, but do we need the extra top margin?

Comment (omarCreativeDev) [reply src/app/features/select-workplace/select-workplace.component.html]: Without the extra margin class the ui doesn't match the design : )

File: src/app/features/select-workplace/select-workplace.component.ts [hunk 2]
```diff
@@ -1,201 +1,131 @@
...
-    data.locationdata[0]['isRegulated'] = false;
+  private getPostalCode(): string {
+    return this.registration.locationdata[0].postalCode;
   }
 
-  workplaceNotFound() {
-    this.addressPostcode = this.registration.locationdata[0].postalCode;
-
-    this.subscriptions.add(
-      this._registrationService.getAddressByPostCode(this.addressPostcode).subscribe(
```
Comment (omarCreativeDev) [inline src/app/features/select-workplace/select-workplace.component.ts]: I removed this api call as I feel its better placed in the next route `/select-workplace-address` at `ngOnit()`

File: src/app/features/select-workplace/select-workplace.component.ts [hunk 3]
```diff
@@ -1,201 +1,131 @@
...
+          this.errorSummaryService.scrollToErrorSummary();
         },
-        () => {
-          console.log('Get location by postcode complete');
-        }
       )
     );
   }
 
-  updateSectionNumbers(data) {
```
Comment (omarCreativeDev) [inline src/app/features/select-workplace/select-workplace.component.ts]: section numbers have been removed from the registration flow. the below code has therefore been removed

**Comment by omarCreativeDev** [comment]
[select workplace page](https://trello.com/c/oh5NcUrg/431-select-workplace-page)

**Pull request closed without merging**